Ux enhancement banner

alexandercyu opened the pull request:
Copied banner from climatedata.ca
*Update 2024-03-27*
- Added space between quickstart menu items
- Added underline to quickstart title

mishaschwartz commented on static/css/daccs-css.css:
```diff
@@ -168,13 +168,97 @@
+    background-attachment: fixed;
+}
+
+.opacity-30 {
```
unused?

alexandercyu replied:
Removed

mishaschwartz commented on static/css/daccs-css.css:
```diff
@@ -168,13 +168,97 @@
+    color: #fff !important;
+}
+
+.bg{
```
unused?

alexandercyu replied:
"bg" class put back in for the static image that is used for smaller screen sizes.

mishaschwartz commented on static/css/daccs-css.css:
```diff
@@ -168,13 +168,97 @@
+    z-index: 1;
+}
+
+.bg-attachment-fixed {
```
unused?

alexandercyu replied:
Removed

alexandercyu replied:
"bg-attachment-fixed" put back in for static image used for smaller screen sizes.

mishaschwartz commented on static/css/daccs-css.css:
```diff
@@ -168,13 +168,97 @@
+    z-index: -100;
+}
+
+.banner-video-overlay{
```
unused?

alexandercyu replied:
May keep this for a static image or overlay in place of video for smaller screen sizes.

alexandercyu replied:
Class is kept in for video.

mishaschwartz commented on static/css/daccs-css.css:
```diff
@@ -168,13 +168,97 @@
     height: 20%;
 }
 
+.banner-video {
```
unused?

alexandercyu replied:
May keep this for a static image or overlay in place of video for smaller screen sizes.

alexandercyu replied:
Removed

mishaschwartz commented on static/css/daccs-css.css:
```diff
@@ -149,8 +149,8 @@
+    background-color: rgba(0.0, 0.0, 0.0, 0.25);
 }
 
 .banner-title-container{
```
unused?

alexandercyu replied:
Removed

mishaschwartz commented on static/video/pexels-mikhail-nilov-8318649 (1440p).mp4:
when we upload pictures and videos can you please name them something that makes sense for the project? Just keeping the default name that you get when you downloaded it makes it really confusing to keep track of what these files are for. I'm actually going to make an issue to clean this up since there's a ton of images now in this project that are named very bizarrely.

alexandercyu replied:
Banner and video renamed.

mishaschwartz commented on templates/site/index.html:
```diff
@@ -7,7 +7,7 @@
                     "Marble is the bridge between intricate climate data and the people who seek to
                     understand it, fostering a community of inquiry and discovery in the realm of climate science."] %}
-
+{% set banner_video = "video/pexels-mikhail-nilov-8318649 (1440p).mp4" %}
```
Since this is the only page that sets this variable, all other pages have a plain grey block at the top.
I think we want there to be no banner/video on pages other than home, right?

alexandercyu replied:
I'll leave the decision to Shruti.

mishaschwartz replied:
@ShrutiK100

alexandercyu replied:
Shruti said all other pages will have a static banner image.  An overlay (the gray block) may still be needed, depending on the colours in the static image.

mishaschwartz commented on static/css/daccs-css.css:
There's actually a bunch of classes that are not longer used in this file. As well as the one's marked below I also see:
- banner-text-background
- banner-image-overlay
- banner-caption
- banner (as an id, not a class)

Please look through this file and remove unused css code

alexandercyu replied:
Removed these classes and id.

alexandercyu commented on templates/partials/banner.html:
```diff
@@ -1,15 +1,39 @@
-        <div class="d-flex align-items-center justify-content-center banner-slogan">
-            {{ page_slogan|default("") }}
-        </div>
+<!--<script src="//unpkg.com/globe.gl"></script>-->
```
Removed the commented line

alexandercyu commented on templates/partials/banner.html:
```diff
@@ -1,15 +1,39 @@
+
+<section id="hero" class="page-section first-section text-white">
+
+    <div id="home-video" class="d-none d-lg-block">
```
The video is supposed to disappear on mobile screen sizes.  I will add an overlay or a static image.

mishaschwartz replied:
A static image would be best

alexandercyu replied:
Added  a placeholder image.  We'll see what Shruti and Steve think about it.

alexandercyu commented on static/css/daccs-css.css:
```diff
@@ -168,13 +168,97 @@
+    backdrop-filter: blur(50px);
+}
+
+.hero-menu a {
```
Removed the underlines and bullets in list element links.

alexandercyu commented on templates/partials/banner.html:
```diff
@@ -1,15 +1,39 @@
+             <div class="row align-items-center">
+
+               <div class="col-10 offset-1 col-sm-8 offset-sm-2 col-lg-6 offset-lg-1">
+                   <h1><span id="banner-title" class="gradient-text banner-title">{{ page_title }}</span></h1>
```
Gradient text removed.

mishaschwartz commented on static/video/banner-video.mp4:
This file is still too big I think.

I'd remove the audio track:

```
ffmpeg -i banner-video.mp4 -map 0 -map -0:a -c copy banner-video-no-audio.mp4
```

and compress it by increasing the crf value:

```
ffmpeg -i banner-video-no-audio.mp4 -crf 28 banner-video-crf33.mp4
```

I'd start at 28 or so and then increase the value until the video quality gets noticeably poor

mishaschwartz replied:
I'd also consider dropping the last clip of the causeway. It's not as climate-y

alexandercyu replied:
The audio tracks for the videos are empty to begin with.

alexandercyu commented:
*Update 2024-03-27*
- Added space between quickstart menu items
- Added underline to quickstart title

mishaschwartz reviewed (changes requested):
There's a lot here that doesn't match the design document (see FIgma):

- the text in the quickstart menu is different
- the title in the quickstart menu doesn't align
- there's a subtitle under Marble
- Marble isn't capitalized

alexandercyu commented:
> There's a lot here that doesn't match the design document (see FIgma):
> 
>     * the text in the quickstart menu is different
> 
>     * the title in the quickstart menu doesn't align
> 
>     * there's a subtitle under Marble
> 
>     * Marble isn't capitalized

- Fixed text in quick start menu
- Fixed alignment of quick start menu title and menu items
- Removed subtitle
- Made Marble title all capitalized
